src/pybind/mgr/rgw/module.py

```diff
@@ -134,8 +180,7 @@ def _cmd_rgw_realm_new_zone_creds(self,
         return HandleCommandResult(retval=retval, stdout=out, stderr=err)
 
     @CLICommand('rgw realm zone-creds remove', perm='rw')
-    def _cmd_rgw_realm_rm_zone_creds(self,
-                                 realm_token : Optional[str] = None):
+    def _cmd_rgw_realm_rm_zone_creds(self, realm_token: Optional[str] = None):
```

Comment:
what's the use case for removing the zone credentials?

Reply:
Same as previous. This is legacy non tested code, I'd suggest removing these commands at least from the docs. We can include them back once they are well tested.